src/napari_micromanager/_gui_objects/_toolbar.py

```diff
@@ -394,3 +191,151 @@ def _add_dock_widget(
             dock_wdg._close_btn = False
         dock_wdg.setFloating(floating)
         return dock_wdg
+
+
+# -------------- Toolbars --------------------
+
+
+class MMToolBar(QToolBar):
+    def __init__(
+        self, title: str, parent: QWidget = None, gb: QGroupBox | None = None
+    ) -> None:
```

Comment:
the `ConfigurationWidget()` was the last that had a QGroupBox. I removed it in https://github.com/pymmcore-plus/pymmcore-widgets/pull/185. after merging we can change this to:
```suggestion
class MMToolBar(QToolBar):
     def __init__(self, title: str, parent: QWidget = None) -> None:
```